recipes/sentry-native/all/conanfile.py

```diff
@@ -40,7 +40,7 @@ class SentryNativeConan(ConanFile):
     default_options = {
         "shared": False,
         "fPIC": True,
-        "backend": "inproc",  # overwritten in config_options
+        "backend": "crashpad",  # overwritten in config_options
```

Comment:
what happened to inproc, is no longer supported after 0.5.x ?

Reply:
I changed the logic so that `crashpad` is the default backend but maybe I make it wrong. From what I read from the previous logic, `inproc` was the default backend but it was overriden in the default option. After reading the upstream repository [CMakeLists.txt](https://github.com/getsentry/sentry-native/blob/master/CMakeLists.txt) I realized that I should probably copy the logic from it.